buildstockbatch/cloud/docker_base.py

```diff
@@ -43,3 +55,131 @@ def docker_image(self):
     @property
     def weather_dir(self):
         return self._weather_dir
+
+    def prep_batches(self, tmppath):
+        """
+        Prepare batches of samples to be uploaded and run in the cloud.
+
+        This will:
+            - Perform the sampling
+            - Split the samples into (at most) self.batch_array_size batches
+            - Collect and package and the required assets, including weather
+              files, and write them to tmppath.
+
+        self.weather_dir must exist before calling this method. This is where weather files are stored temporarily.
```

Comment:
Does this need to be a property? It doesn't seem to be used by anything else and would be simpler and more straightforward to just pass in as a parameter.

Reply:
I thought so too, but it turns out it's used in the base class [here](https://github.com/rewiringamerica/buildstockbatch/blob/661f66abc1e78faf47c9ba56c07ba0918278a15c/buildstockbatch/base.py#L102).

Reply:
ah, ok :grimacing: